gateway-api.yaml

```
@@ -0,0 +1,124 @@
+openapi: 3.0.0
+info:
+  title: Digital Classroom Gateway API
+  description: API to control the digital classroom application from the gateway service. This APi is based on the Big Blue Button API.
+  version: 0.0.1
+
+tags:
+  - name: Create Room
+    description: Creating a new digital classroom
+servers:
+  - url: 'https://localhost:8080/api/v1'
```

```suggestion
  - url: 'https://localhost:8085/api/v1'
```

The README says to use `https://localhost:8085`, shouldn't that be here too?

---

You‘re right. I just haven‘t updated this for a while. Thank you.